src/topic/components/Topic.js

```diff
@@ -3,16 +3,22 @@ import TopicDescription from './TopicDescription';
 import TrainersList from './TrainersList';
 import styled from 'styled-components';
 import Participants from './Participants';
+import PropTypes from 'prop-types';
```

Comment:
eslint doesn't send a warning here? should be on top with styled-components (picky, but I'm just curious)

Reply:
No a single warning. Could you please provide a source why it should be on top?